components/esp_websocket_client/include/esp_websocket_client.h

```diff
@@ -193,6 +194,19 @@ int esp_websocket_client_send_bin(esp_websocket_client_handle_t client, const ch
  */
 int esp_websocket_client_send_text(esp_websocket_client_handle_t client, const char *data, int len, TickType_t timeout);
 
+/**
+ * @brief      Write opcode data to the WebSocket connection
+ *
+ * @param[in]  client  The client
+ * @param[in]  opcode  The opcode
+ * @param[in]  data    The data
+ *
```

Comment:
Please complete the docs:
```suggestion
 * @param[in]  len  ....
 * @param[in]  timeout ...
 *
```